src/Parser/RetributionPaladin/Modules/PaladinCore/BlessingOfTheAshbringer.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import SPELLS from 'common/SPELLS';
+import SpellLink from 'common/SpellLink';
+import { formatPercentage } from 'common/format';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import Module from 'Parser/Core/Module';
+
+const debug = false;
+
+class BlessingOfTheAshbringer extends Module {
+	static dependencies = {
+		combatants: Combatants,
+	};
+
+
+	//Blessing of the Ashbringer = BotA
+	botaUptime = 0;
```

Comment:
How about naming this just `uptime`? The context of the module is `BlessingOfTheAshbringer` so, specifying that it's about bota again is unnecessary (you wouldn't do `car.carSpeed`).